tools/test-proxy/Azure.Sdk.Tools.TestProxy/Startup.cs

```diff
@@ -37,13 +42,48 @@ private static string resolveRepoLocation(string storageLocation = null)
             return storageLocation ?? envValue ?? Directory.GetCurrentDirectory();
         }
 
+        private static string[] resolvePorts(int inputHttpPort, int inputHttpsPort)
+        {
+            var httpFinalPort = HttpDefaultPort;
+            var httpsFinalPort = HttpsDefaultPort;
+            var portConfigFromEnv = Environment.GetEnvironmentVariable("ASPNETCORE_URLS");
+            if (portConfigFromEnv != null)
+            {
+                foreach (var portConfig in portConfigFromEnv.Split(';'))
```

Comment:
If someone set's `ASPNETCORE_URLS`, we should pass that value directly to `UseUrls()`. You may have to split by `;` if we have to, though I have a hunch that UseURLs will be fine with `http://0.0.0.0:5000;https://0.0.0.0:5001` for example.

Reason being, we need to map more than just the port right. If we were to set ASPNETCORE_URLS [like this for instance](https://github.com/Azure/azure-sdk-tools/blob/9d05b726db5d0d632709e788d2d27102b89b4c2a/tools/test-proxy/docker/dockerfile#L18)

We are binding to `0.0.0.0` instead of just `localhost:5000` or `*:5000`. This is due to the unique constraints of hosting within docker and talking to the loopback device.